src/ui/Views/Courses/Index.cshtml

```diff
@@ -2,11 +2,24 @@
 
 @{
     Layout = "_Layout";
+    ViewBag.Title = Model.Courses.OrganisationName;
+}
+@if (Model.MultipleOrganisations)
```

Comment:
With regards to this you may want to look at this implementation

https://github.com/DFE-Digital/search-and-compare-ui/blob/23738eb962d982007a01d63e6558a1cf1ee2e5bb/src/Views/Results/Index.cshtml#L46

Reply:
That's fine